front/components/data_source/MainTab.jsx

```diff
@@ -16,6 +20,16 @@ export default function MainTab({ dataSource, currentTab, owner, readOnly }) {
         />
       ),
     },
+    {
```

Comment:
We want the user to be logged in to display `Search` since we need to at least have a chance at using their credentials if they exist. If not logged in there have no credentials and search is guaranteed to not work.

Reply:
👍 I added the tab to the list only if the user is logged in. LMK if that's what you had in mind.